### ISSUE TITLE
Fullscreen map does not load all tiles

### ISSUE THREAD

**Issue by killakalle** [theCrag/website#3895]
**What happened?**
Full screen map does not load all map tiles.

Open any crag, go to map and extend it to full screen. In my case it does not load all the map tiles. When I move around it always seems to load only tiles in the upper left corner. See my linked video.

Maybe the algorithm still assumes I am on the small map rectangle without extending to fullscreen?

Example url(s) to reproduce the problem:

Ideally also provide screenshots:
![image](https://user-images.githubusercontent.com/30682796/116618468-6821a000-a93f-11eb-827c-a484baaba0d4.png)

Video showing the issue:
https://youtu.be/FEra8d7QrwY

**What you expected:**

* [ ] Map loads all tiles in fullscreen

**Comment by rouletout** [comment]
Just checked on really poor network and I don't have the issue in any of the layers available. Is it a specific layer? I saw that sometimes with OSM (Cached) but not with others.

**Comment by killakalle** [comment]
This might be Safari issue. On Chrome everything works for me, too.
However, I can reproduce this issue using any map layer on two different Macbooks with Safari

**Comment by rouletout** [comment]
Yes, this is on Safari only. I tested Chrome, Firefox and Vivaldi, they all work correctly - only Safari shows this behaviour. I don't think that we will do anything about it for now.
